framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/scenario/builder/WorkbenchKieServerDatabasePersistentScenarioBuilderImpl.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2018 JBoss by Red Hat.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.kie.cloud.openshift.scenario.builder;
+
+import java.util.HashMap;
+import java.util.Map;
+import org.kie.cloud.api.deployment.constants.DeploymentConstants;
+import org.kie.cloud.api.scenario.WorkbenchKieServerDatabasePersistentScenario;
+import org.kie.cloud.api.scenario.builder.WorkbenchKieServerDatabasePersistentScenarioBuilder;
+import org.kie.cloud.openshift.constants.OpenShiftTemplateConstants;
+import org.kie.cloud.openshift.scenario.WorkbenchKieServerDatabasePersistentScenarioImpl;
+
+/**
+ *
+ * @author jschwan
+ */
+public class WorkbenchKieServerDatabasePersistentScenarioBuilderImpl implements WorkbenchKieServerDatabasePersistentScenarioBuilder {
+
+    private Map<String, String> envVariables;
+
+    public WorkbenchKieServerDatabasePersistentScenarioBuilderImpl() {
+        this.envVariables = new HashMap<String, String>();
```

Comment:
Please initialize in filed initializer (`private Map<String, String> envVariables = new HashMap<String, String>();`) and remove the superfluous `this.` in constructor to keep consistency with other scenario builders.